index.html

```diff
@@ -246,7 +246,7 @@ <h2><dfn>measure()</dfn> method</h2>
               <li>Otherwise, let <var>start time</var> be <code>0</code>.</li>
             </ol>
           </li>
-          <li>Create a new <a>PerformanceMeasure</a> object (<var>entry</var>).</li>
+          <li>Create a new <a>PerformanceMeasure</a> object (<var>entry</var>) with the <a data-cite="ECMA-262#current-realm">current realm</a>.</li>

```

Comment:
Rather than referring to the current relm. You can simply refer to the realm of the context object's global object since the context object has an associated global object.

Reply:
Sure. context object = this per https://dom.spec.whatwg.org/#context-object. Also I think I can just reference its relevant realm.